src/Pages/3DPrintingConsole/3DConsole.js

```diff
@@ -88,7 +88,7 @@ export default class PrintConsole3D extends React.Component {
     }
   }
 
-  // simply filter array by name
+  // simply filter array by name!
```

Comment:
remove this line all together

Reply:
oops we were using this line to test grpc issue, and left it